Assets/Scripts/PlayerOrbitMoveControl.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerOrbitMoveControl : MonoBehaviour
+{
+    
+    public Transform target;
+    public float distance = 20.0f;
+    public float xSpeed = 40.0f;
+    public float ySpeed = 80.0f;
+ 
+    public float yMinLimit = -50f;
+    public float yMaxLimit = 100f;
+ 
+    public float distanceMin = 10f;
+    public float distanceMax = 40f;
+
+    
+    float x = 0.0f;
+    float y = 0.0f;
+    // Start is called before the first frame update
+    void Start()
+    {
+        Vector3 angles = transform.eulerAngles;
+        x = angles.y;
+        y = angles.x;
+    }
+
+    void LateUpdate () 
+    {
+        #if UNITY_EDITOR || UNITY_STANDALONE || UNITY_WEBGL
+        if (target && Input.GetMouseButton(0)) 

```

Comment:
`target` must be some unity game object available to this monobehavior -- is it the click target?

Reply:
It's the object that is in the scene that is the target / center for the orbital movement - probably easiest to demo this later!